src/api/samples.js

```diff
@@ -19,15 +19,17 @@ export async function getAllDetailedSamples({
   accessionCodes,
   orderBy,
   offset,
-  limit
+  limit,
+  filter_by
 }) {
   if (accessionCodes && accessionCodes.length) {
     let { results } = await Ajax.get('/samples/', {
       // send the accession codes as a string, otherwise they will be converted to an array parameter
       accession_codes: accessionCodes.join(','),
       offset,
       limit,
-      order_by: orderBy
+      order_by: orderBy,
+      ...(filter_by ? { filter_by } : {})
```

Comment:
I think this line can be simplified into:
```...filter_by```
because when `filter_by` is null, `...filter_by` will be blank. You can test it.

Reply:
haha really!? nice tip!! I'll update it